File: app/src/main/java/io/github/ovso/massage/main/f_acupoints/adapter/ImagesViewHolder.java
package io.github.ovso.massage.main.f_acupoints.adapter;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import butterknife.BindView;
import io.github.ovso.massage.R;
import io.github.ovso.massage.framework.adapter.BaseRecyclerAdapter;

public class ImagesViewHolder extends BaseRecyclerAdapter.BaseViewHolder {

    @BindView(R.id.doc_url_textview)
    TextView docUrlTextview;
    @BindView(R.id.imageview)
    ImageView imageview;
    public ImagesViewHolder(View itemView) {
        super(itemView);
    }
}

File: app/src/main/java/io/github/ovso/massage/main/f_acupoints/adapter/ImagesViewHolder.kt
package io.github.ovso.massage.main.f_acupoints.adapter

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import io.github.ovso.massage.R
import io.github.ovso.massage.framework.adapter.BaseRecyclerAdapter

class ImagesViewHolder(itemView: View) : BaseRecyclerAdapter.BaseViewHolder(itemView) {
    @JvmField
    var docUrlTextview: TextView = itemView.findViewById<View>(R.id.doc_url_textview) as TextView

    @JvmField
    var imageview: ImageView = itemView.findViewById<View>(R.id.imageview) as ImageView

}